content/pt-br/docs/reference/scheduling/policies.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Políticas do escalonador
+content_type: concept
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
+weight: 30
+---
+
+<!-- overview -->
+
+Nas versões anteriores à v1.23 do Kubernetes, a política do escalonador pode ser usada para especificar os processos *predicates* e *priorities*. Por exemplo, você pode definir a política do escalonador rodando os seguintes comandos:
+```
+ `kube-scheduler --policy-config-file <filename>` 
+ ou
+  `kube-scheduler --policy-configmap <ConfigMap>`.
+ ```
```

Comment:
This block is rendering really strangely. The fenced code block should not be here (` ``` `), as we only want the commands to be rendered as code (and not the word "ou").